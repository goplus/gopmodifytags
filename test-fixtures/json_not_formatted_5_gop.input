type x struct {





	Foo int
	bar string

}
