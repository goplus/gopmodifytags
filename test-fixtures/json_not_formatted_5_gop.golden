{
  "start": 1,
  "end": 10,
  "lines": [
    "type x struct {",
    "",
    "",
    "",
    "",
    "",
    " Foo int `json:\"foo\"`",
    "\tbar string `json:\"bar\"`",
    "",
    "}"
  ]
}